controller/src/beerocks/master/db/node.h

```diff
@@ -28,6 +30,22 @@ typedef struct {
     bool backhaul_vap;
 } sVapElement;
 
+
+
+typedef struct {
+    
+    //TX link metrics related data
+    sMacAddr mac_of_an_interface_in_the_receiving_al;
+    sMacAddr mac_of_an_interface_in_the_neighbor_al;
+    std::list<ieee1905_1::tlvTransmitterLinkMetric::sLinkMetricInfo> PairMetricInfo;    
+    
+    //RX link metrics related data
+    sMacAddr mac_of_an_interface_in_the_receiving_al;
+    sMacAddr mac_of_an_interface_in_the_neighbor_al;
+    std::list<ieee1905_1::tlvReceiverLinkMetric::sLinkMetricInfo> PairMetricInfo;    
+    
+} sTxRxLinkMetricsData;
```

Comment:
Long term, I think we should decompose the link metric data into the information which is actually useful.

For now, however, we can stay pretty close to the TLV structures.
Therefore, I think it should be
```cpp
typedef struct {
    sMacAddr al_mac_of_the_device_that_reports;
    sMacAddr al_mac_of_the_device_that_is_reported;
   std::list<ieee1905_1::tlvTransmitterLinkMetric::sInterfacePairInfo> transmitterLinkMetrics;
   std::list<ieee1905_1::tlvReceiverLinkMetric::sInterfacePairInfo> receiverLinkMetrics;
} sLinkMetricsData;
```

Also, please use doxygen comments.